src/app/static/src/app/store/translations/locales.ts

```diff
@@ -329,6 +347,8 @@ const fr: Partial<Translations> = {
     bar: "Barre",
     badUsernamePassword: "Le nom d'utilisateur ou le mot de passe est incorrect",
     browse: "Parcourir",
```

Comment:
Maybe this should be changed to Chercher for consistency? Rich suggested that Parcourir should be changed to Chercher: "I don’t think this is right. This is in the sense of “look in ADR” right? Chercher ADR might be better?"

Reply:
So that's the text on the button? Yeah, should consistent here, but didn't that key already exist and had another use (or uses) elsewhere? If so, should check that Chercher makes sense for those. Browse and Search (chercher) do seem like slightly different concepts to me, so other uses of Parcourir might be valid? Ask Rich!